packages/runtime-common/matrix-constants.ts

```diff
@@ -2,6 +2,7 @@ export const APP_BOXEL_CARDFRAGMENT_MSGTYPE = 'app.boxel.cardFragment';
 export const APP_BOXEL_MESSAGE_MSGTYPE = 'app.boxel.message';
 export const APP_BOXEL_COMMAND_MSGTYPE = 'app.boxel.command';
 export const APP_BOXEL_CARD_FORMAT = 'app.boxel.card';
+export const APP_BOXEL_COMMAND_RESULT_EVENT_TYPE = 'app.boxel.commandResult';
 export const APP_BOXEL_COMMAND_RESULT_MSGTYPE = 'app.boxel.commandResult';
```

Comment:
Do we still need a msgtype?

If we do, should we use this to distinguish between filled and undefined results? The event type says it's a command result, but the msgtype could be how to parse that (e.g. linked card or nothing). Notably we don't seem to use the msgtype on the bot side.

Reply:
Updating this...